cluster-baseline-settings/akv-secrets-store-csi.yaml

```diff
@@ -310,6 +431,50 @@ spec:
             path: /etc/kubernetes/secrets-store-csi-providers
             type: DirectoryOrCreate
 ---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
I think (IIRC) we grouped all `ClusterRoles` at the top. That was a nice way to see them all together and organized.

Reply:
Ah, I thought it was <secret store csi bits> followed by <azure keyvault bits>.  I'll move this role to the top though, that'll be just fine.

Reply:
Good catch, didn't even notice that.